mips/exc.S

```diff
@@ -70,9 +71,19 @@ user_exc_enter:
         SAVE_REG($t2, BADVADDR, $k0)
         SAVE_REG($t3, PC, $k0)
 
-        # TODO: switch to kernel SP!
+        # Load kernel global pointer.
         la      $gp, _gp
 
+        # Load kernel stack pointer (always stack bottom ?)
+        LOAD_PCPU($t0)
+        lw      $t0, PCPU_CURTHREAD($t0)
+        lw      $sp, TD_KSTACK+STK_BASE($t0)
+        lw      $t1, TD_KSTACK+STK_SIZE($t0)
+        add     $sp, $t1
+
+        # No exeception frame so set td_frame to NULL.
+        sw      $0, TD_KFRAME($t0)
```

Comment:
Is `td_kframe` useful at all if `exc_frame_t` is passed to exception / interrupt handler directly ?